localisation/00_zuwc_united_localisation.csv

```diff
@@ -2985,15 +2976,29 @@ repentance_ask;The [Root.RelHead.GetTitle] will hear our plea for salvation.;Le
 EVTDESC39253;The [Root.RelHead.GetTitle] has agreed to lift the excommunication against me!;Le [Root.RelHead.GetTitle] a accept� de lever mon excommunication!;Der [Root.RelHead.GetTitle] hat zugestimmt, meine Exkommunikation aufzuheben!;;�El [Root.RelHead.GetTitle] ha accedido a revocar la excomuni�n contra mi!;;;;;;;;;x
 can_not_change_culture_evil_true_religion_trigger_negative_tooltip;�Y[From.Religion.GetName] [From.Culture.GetName]�! characters can NOT change culture;;;;;;;;;;;;;x
 not_importrant_place_tooltip;Is NOT an important place in event chains;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+quest_control_mind_smart;Mind Control Clever Man or Woman;;;;;;;;;;;;;x
+quest_control_mind_strong;Mind Control Sturdy Man or Woman;;;;;;;;;;;;;x
+quest_control_mind_fair;Mind Control Attractive Man or Woman;;;;;;;;;;;;;x
```

Comment:
I'd use "Individual" e.g. "Clever Individual" to maintain your need for narrative sense

Reply:
Implemented!